src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx

```diff
@@ -11,25 +12,7 @@ For setting up automatic controls over how your New Relic users are added to New
 * <DNT>**SAML SSO**</DNT>: this allows your users to use a single sign-on (SSO) identity provider service to log in to New Relic, as opposed to using the default email + password. Requires a [paid edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions).
 * <DNT>**SCIM provisioning**</DNT>, also referred to as automated user management: SCIM provisioning allows organizations to use their identity provider service to automate how their users are added to and updated in New Relic. Organizations using SCIM will almost always use SAML SSO. Requires [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions).
 
-For information on how to enable SAML and SCIM, choose your user model below. If you're not sure which you're on, see [Overview of user models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models).
+For information on how to enable SAML and SCIM, you'll use the [authentication domain UI](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) to set up SAML and SCIM. Note that Okta, OneLogin, and Azure have dedicated New Relic apps: If you use those, you must choose the one called **New Relic by organization**, and not **New Relic by account**.
```

Comment:
I'm not sure if the If in the latter part of the sentence should be capitalized?

Reply:
@jeff-colucci I think we're only capitalizing following a colon if there's a complete sentence.